scopesim/effects/ter_curves_utils.py

```diff
@@ -386,6 +383,8 @@ def combine_two_spectra(spec_a, spec_b, action, wave_min, wave_max):
         # plt.show()
     elif "add" in action.lower():
         spec_c = spec_a(wave) + spec_b(wave)
+    else:
+        raise ValueError("action %s unknown", action)
```

Comment:
```suggestion
        raise ValueError(f"action {action} unknown")
```
The "lazy" formatting is just for logging functions, not for actual exception raising. The way it was done would not actually produce a correct error msg I think.